sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ImportTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    /// <summary>
+    /// Tests for DigitalTwinServiceClient methods dealing with Digital Twin Import operations.
+    /// </summary>
+    public class ImportTests : E2eTestBase
+    {
+        public ImportTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task Import_Lifecycle()
```

Comment:
@abhinav-ghai any other tests we want to add here? Kind of went off what was in the ModelTests.cs file, but we can add some more as well.

Reply:
discussed offline and added more